marcosberges.txt

```diff
@@ -0,0 +1,3 @@
+name: marcos berges
```

Comment:
And, as a last step, can you remove this file? That way we don't pollute the root directory

Reply:
I thought It would delete here, since i deleted it locally. Could you tell me how to delete it?

Reply:
It's a bit counter intuitive, but after deleting it you have to `git add marcosberges.txt` (yes, `add` in this case tells git that it's been removed) and then commit the result.

Reply:
Done! Thanks for the help.